maven-release-manager/src/test/java/org/apache/maven/shared/release/phase/AbstractReleaseTestCase.java

```diff
@@ -213,7 +213,13 @@ public FileVisitResult visitFile( Path file, BasicFileAttributes attrs )
         for ( MavenProject project  : reactorProjects )
         {
             MavenProject resolvedProject = projectBuilder.build( project.getFile(), buildingRequest ).getProject();
-            
+
+            // FIXME ... setDependencyArtifacts -  set direct dependencies ...
+            // but in org.apache.maven.shared.release.phase.GenerateReleasePomsPhase.createReleaseDependencies
+            // getArtifacts is used ...
+
+            // so we probably also need call setResolvedArtifacts
+
```

Comment:
Probable cause of ignore tests in AbstractRewritingReleasePhaseTestCase

Reply:
For example, I have hard time to understand why release needs to resolve anything at all...

Reply:
And why we need use `setDependencyArtifacts` directly ... 
code is also strange for me we check:
```
if ( project.getDependencyArtifacts() == null ) 
```
and next on different object
```
resolvedProject.setDependencyArtifacts(....)
```

I will check what happens if I call 

```
resolvedProject.setResolvedArtifacts( resolvedProject.getDependencyArtifacts() )
```
of course I don't understand it very well